apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -114,6 +114,23 @@ public static Integer parseInteger(String intText) {
     }
   }
 
+  /**
+   * @param longText the number string to parse
+   * @return the specified text parsed into an {@link Long}
+   */
+  public static Long parseLong(String longText) {
+    /*
+     * Might seem silly to pull this out, but it makes the code a bit easier
+     * to read, and ensures that this parsing is standardized.
+     */
```

Comment:
🍰  If we're going to have a block comment describing the method at the start, why not just throw it in the javadoc?

Reply:
This got copy-pasted from another method above and I think I am going to throw it away as it provides practically no value IMO and does not seem like useful javadoc to me either.